make_output_unet_cmd.py

```diff
@@ -99,7 +99,11 @@ def divide_batch(l, n):
     for ii,fname in enumerate(files):
         print(f"PROGRESS: {ii}/{nfiles}")
         fname = fname.strip()
-        newfname_class = "%s/%s_pred.png" % (OUTPUT_DIR, os.path.basename(fname)[0:-4])
+        
+        # change file extension to png
+        newfname_class = '.'.join(os.path.basename(fname).split('.')[:-1]) + '_pred.png'
```

Comment:
This is still a bit hackish, the correct way to do this in python:

https://docs.python.org/3/library/os.path.html#os.path.splitext

which should be able to simplify this line a bit. want to quickly revise?

Reply:
Super - thank you! A much more stable fix. Neat!